platform/podjs/src/browserPod.ts

```diff
@@ -13,7 +13,7 @@ import { EncodingOptions, Stats, Entry } from "@polypoly-eu/pod-api";
 import { dataFactory } from "@polypoly-eu/rdf";
 import * as RDF from "rdf-js";
 import * as zip from "@zip.js/zip.js";
-import { Manifest, readManifest } from "@polypoly-eu/manifest-parser";
+import { Manifest, readManifest } from "./manifest-parser/src/manifest";
```

Comment:
Do we need to keep it in its own independent directory? At any rate, keeping the `src` is not redundant?

Reply:
there was no clear requirement on the jira description so I wasn't sure for that - so we want it next to `browserPod` then?

Reply:
It's just a file, at the end of the day. It seems simpler.